gamedata/scripts/z_auto_looter.script

```diff
@@ -435,6 +445,7 @@ end
 
 local function load_state(mdata)
     favorite_itms = mdata.rax_favorite_itms or favorite_itms
+	junk_itms = mdata.rax_junk_itms or junk_itms
```

Comment:
The default value of `junk_itms` is missing. Define it near the fav items as an empty table please, as otherwise it would cause crash when there's no Raven's Sorting Plus